code/modules/projectiles/guns/energy/special.dm

```diff
@@ -95,7 +95,7 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/bolt)
 	weapon_weight = WEAPON_LIGHT
 	obj_flags = 0
-	overheat_time = 20
+	overheat_time = 20 SECONDS
```

Comment:
```suggestion
	overheat_time = 2 SECONDS
```

Reply:
20 seconds is far longer than literally any other gun, if you want it to be "skill intensive" make it like 4 seconds or something

Reply:
literally no other gun is basically a ranged sleepy pen either and having the cooldown be way shorter would make it so you could get stupid levels of radiation on someone in 5 seconds, and it would be extremely easy to chain ebow someone because they're slowed after you shoot them, making the debuffs last ages.

Reply:
hmm, fair enough, if it ends up being a problem a future pr can be made